app/components/InstantMessage/ImMain/ImHistory/History.jsx

```diff
@@ -34,18 +34,21 @@ module.exports = React.createClass({
         return {
             messages: [],
             upperThreshold: 100,
-            messageSuites : Immutable.List.of()
+            messageSuites : MessageStore.getCurrentChannelMessageSuites()
         }
     },
 
     componentDidMount() {
         MessageStore.on(IMConstant.EVENTS.RECEIVE_MESSAGE, this._onReceiveMessage);
         MessageStore.on(IMConstant.EVENTS.RECEIVE_NEW_MESSAGE, this._onReceiveNewMessage);
+
+        var node = this.getDOMNode();
+        node.scrollTop = node.scrollHeight;
     },
 
     componentWillUnmount() {
         MessageStore.removeListener(IMConstant.EVENTS.RECEIVE_MESSAGE, this._onReceiveMessage);
-        MessageStore.on(IMConstant.EVENTS.RECEIVE_NEW_MESSAGE, this._onReceiveNewMessage);
+        MessageStore.removeListener(IMConstant.EVENTS.RECEIVE_NEW_MESSAGE, this._onReceiveNewMessage);
```

Comment:
Since `_onReceiveNewMessage` is no longer used, I think we can remove this lisenter, and the EVENTS type?